tools/gen_docs.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
```

Comment:
If its all the same, can we use a python script instead?

Reply:
Sure.  I mean, I would do a Dart script, but I wasn't sure about whether dart was always available in the CI here.

Reply:
Don't know about Dart. I meant Python.

Reply:
Your call though. The bash stuff isn't all that portable should we want to run this or parts thereof on Windows.

Reply:
Swapped it out for a python script.